libpod/runtime.go

```diff
@@ -118,10 +126,10 @@ type RuntimeConfig struct {
 	// cause conflicts in containers/storage
 	// As such this is not exposed via the config file
 	StateType RuntimeStateStore `toml:"-"`
-	// RuntimePath is the path to OCI runtime binary for launching
-	// containers
-	// The first path pointing to a valid file will be used
-	RuntimePath []string `toml:"runtime_path"`
+	// OCIRuntime is the OCI runtime to use.
+	OCIRuntime string `toml:"-"`
```

Comment:
Should we expose this in the config so the default runtime can be set?

Reply:
good point, fixed